ct/cmd/root.go

```diff
@@ -63,6 +63,7 @@ func addCommonFlags(flags *pflag.FlagSet) {
 	flags.StringVar(&cfgFile, "config", "", "Config file")
 	flags.String("remote", "origin", "The name of the Git remote used to identify changed charts")
 	flags.String("target-branch", "master", "The name of the target branch used to identify changed charts")
+	flags.String("commit", "HEAD", "The commit used to identify changed charts")
```

Comment:
> from https://github.com/helm/chart-testing/issues/64#issuecomment-507276419: After merging the PR I want to git diff .. with the previous commit `HEAD~1` to be able to detect the changes, test again and finally push the chart to a helm repository.

This is confusing to me. Why wouldn't `HEAD` work in this case?

If there is a use case for this, it is still possible to change `HEAD` by doing a `git checkout <ref>` before running `ct`.